sdk/cognitiveservices/pom.xml

```diff
@@ -14,6 +14,13 @@
   <description>This package contains the parent module of Microsoft Azure Cognitive Services SDK.</description>
   <url>https://github.com/Azure/azure-sdk-for-java</url>
 
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-data-sdk-parent</artifactId>
+    <version>1.0.0</version>
+    <relativePath>../../pom.data.xml</relativePath>
+  </parent>
```

Comment:
As noted on another PR, now that #4289 is filed, I would track it in there rather than eagerly start the transition to a compliant track one here (but not finish or validate it).

Reply:
I think without this change it will fail to build. Some of these track 1 code bases weren't building at all. So this is a minimal change to make sure that the code at least builds so that if someone introduces a change we can block PR completion until it is resolved.

Do you still want me to take it out given that context?

Reply:
I've unlinked the root ```pom.data.xml``` file to see how it goes in the CI system. Ultimately though there may be a case to remove this POM alltogether and go back to the root one, but we need to look through the POM in detail to understand side effects. Commented out for now and if it works I'll delete the lines entirely.